run-spread

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# check dependencies
+if ! snap list snapcraft &>/dev/null; then
+    echo "snapcraft is not installed"
+    exit 1
+fi
+
+# Clean the snaps created in previous runs
+touch test-build
+rm -rf built-snap
+mkdir built-snap
+rm -f snapd_1337.*.snap snapd_1337.*.snap.keep
```

Comment:
I think if one of these doesn't find any matches, both might fail (this is the case on zsh at least, though I know this is `#!/bin/bash`), might be best to put each of these on their own line.

Reply:
done